scripts/postProcessing/hydraulicFractureVerification/HydrofractureSolutions.py

```diff
@@ -91,288 +85,306 @@ def Solutions( self, mu, E, q, KI, time, time_radplots, distanceFromInjection ):
-    
-    def Lk(self,Q0,Ep,t,KIp):
-        return pow( Ep * Q0 * t / KIp , 2.0/3.0)
-        
+    def epsilon_m(self, mup, Ep, t):
+        return pow(mup / (Ep * t), 1.0 / 3.0)
+
+    def Lm(self, Ep, Q0, t, mup):
+        return pow(Ep * pow(Q0, 3) * pow(t, 4) / mup, 1.0 / 6.0)
+
+    def epsilon_k(self, KIp, Ep, Q0, t):
+        return pow(pow(KIp / Ep, 4.0) / (Q0 * t), 1.0 / 3.0)
+
+    def Lk(self, Q0, Ep, t, KIp):
+        return pow(Ep * Q0 * t / KIp, 2.0 / 3.0)
+
     def gamma_m0(self):
         return 0.616
-    
+
     def gamma_k0(self):
-        return 2.0 / pow(math.pi,2.0/3.0) 
+        return 2.0 / pow(math.pi, 2.0 / 3.0)
 
-    def Omega_m0_bar(self, rho ):
-        A0 =  math.sqrt(3.0)
+    def Omega_m0_bar(self, rho):
+        A0 = math.sqrt(3.0)
         A1 = -0.156
-        B1 =  0.0663
-        return A0 * pow( 1.0 - rho * rho , 2.0/3.0 ) \
-             + A1 * pow( 1.0 - rho * rho , 5.0/3.0 ) \
-             + B1 * ( 4.0 * math.sqrt(1-rho*rho) + 2.0*rho*rho*math.log( (1.0-math.sqrt(1-rho*rho))/(1.0+math.sqrt(1-rho*rho)) ) )
+        B1 = 0.0663
+        return (
+            A0 * pow(1.0 - rho * rho, 2.0 / 3.0)
+            + A1 * pow(1.0 - rho * rho, 5.0 / 3.0)
+            + B1
+            * (
+                4.0 * math.sqrt(1 - rho * rho)
+                + 2.0
+                * rho
+                * rho
+                * math.log(
+                    (1.0 - math.sqrt(1 - rho * rho)) / (1.0 + math.sqrt(1 - rho * rho))
+                )
+            )
+        )
 
     def PI_m0(self, rho):
         A1 = 1.61750
         A2 = 0.39650
         B = 0.06858
-        PIStar1 = 1.0 / (3.0 * math.pi) * scipy.special.beta(0.5, 2.0/3.0) * scipy.special.hyp2f1(-1.0/6.0, 1, 0.5, rho*rho)
-        PIStar2 = 3.0 / (20.0 * math.pi) * scipy.special.beta(-1.5, 8.0/3.0) *( 2.0/3.0 * rho*rho * scipy.special.hyp2f1(-1.0/6.0, 2, 1.5, rho*rho) - 0.5* scipy.special.hyp2f1(-7.0/6.0, 1, 0.5, rho*rho) )
-
-        return A1 * PIStar1 + A2 * PIStar2 + B * (2.0-math.pi*abs(rho))
-#        A0 =  math.sqrt(3.0)
-#        A1 = -0.156
-#        B1 =  0.0663
-#        B = 2.58711
-#        return  1.0 / ( 3.0 * math.pi ) * B * A0 * scipy.special.hyp2f1(-1.0/6.0,1,0.5,rho*rho) \
-#              + 10.0/7.0 * A1 * scipy.special.hyp2f1(-7.0/6.0,1,0.5,rho*rho) \
-#              + B1*(2-math.pi*abs(rho))
-
-
-
-
-
+        PIStar1 = (
+            1.0
+            / (3.0 * math.pi)
+            * scipy.special.beta(0.5, 2.0 / 3.0)
+            * scipy.special.hyp2f1(-1.0 / 6.0, 1, 0.5, rho * rho)
+        )
+        PIStar2 = (
+            3.0
+            / (20.0 * math.pi)
+            * scipy.special.beta(-1.5, 8.0 / 3.0)
+            * (
+                2.0
+                / 3.0
+                * rho
+                * rho
+                * scipy.special.hyp2f1(-1.0 / 6.0, 2, 1.5, rho * rho)
+                - 0.5 * scipy.special.hyp2f1(-7.0 / 6.0, 1, 0.5, rho * rho)
+            )
+        )
```

Comment:
🙈

Reply:
That is a bit gross.  Apparently this was adopted into a recent version of pep8: https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator.  We could dial up the line length to eleven, and avoid these issues entirely.

Reply:
Although these lines in particular are a bit annoying, and would improve with a few additional parentheses (I don't like naked divide expressions, as you have to start thinking about order of operations).